Library/Formula/aws-keychain.rb

```diff
@@ -0,0 +1,15 @@
+class AwsKeychain < Formula
+  desc "Uses OS X keychain for storage of AWS credentials."
+  homepage "https://github.com/pda/aws-keychain"
+  url "https://github.com/pda/aws-keychain/archive/v3.0.0.tar.gz"
+  sha256 "3c9882d3b516b629303ca9a045fc50f6eb75fda25cd2452f10c47eda205e051f"
+
+  def install
+    bin.install "aws-keychain"
+  end
+
+  test do
+    # It is not possible to create a new keychain without triggering a prompt.
```

Comment:
A graphical prompt from the built in security command prompting for keychain unlock.
![prompt](https://cloud.githubusercontent.com/assets/983/13751218/7db588b6-e9df-11e5-9881-24ac1c104370.png)